osquery/tables/system/windows/certificates.cpp

```diff
@@ -542,8 +659,29 @@ BOOL WINAPI certEnumSystemStoreLocationsCallback(LPCWSTR storeLocation,
   return TRUE;
 }
 
-QueryData genCerts(QueryContext& context) {
-  QueryData results;
+/// A user's `Personal` certs are stored on disk and not in the registry.
```

Comment:
I want to make sure we are being consistent in the method documentation style. If I recall correctly the pattern in the code base is:
```
/**
  * Content
  */
```

Reply:
Yeah I'd +1 this sentiment throughout your diff also. I believe we're still doing multi-line comments as @theopolis is describing, and `//` for single line comments.